packages/ethereum-contracts/tasks/deploy-cfa-forwarder.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash -eux
+
+# Usage:
+# tasks/deploy-cfa-forwarder.sh <network> <contract-addr>
+#
+# Example:
+# tasks/deploy-cfa-forwarder.sh optimism-goerli 0xcfa132e353cb4e398080b9700609bb008eceb125
+#
+# The invoking account needs to be (co-)owner of the resolver and governance
+#
+# important ENV vars:
+# RELEASE_VERSION, DETERMINISTIC_DEPLOYER_PK, RESOLVER_ADMIN_TYPE, GOVERNANCE_ADMIN_TYPE
+#
+# You can use the npm package vanity-eth to get a deployer account for a given contract address:
+# Example use: npx vanityeth -i cfa1 --contract
+#
+# Note that the value of DETERMINISTIC_DEPLOYER_PK needs to match the given contract-addr.
+# The script will not check this, but fail (at contract verification) if not matching.
+#
+# For optimism the gas estimation doesn't work, requires setting EST_TX_COST.
+# For polygon-mainnet, GAS_PRICE usually needs to be set.
+#
+# On some networks you may need to use override ENV vars for the deploymentto succeed
+
+network=$1
+cfaAddr=$2
+
+# deploy
+#npx truffle exec --network $network scripts/deploy-deterministically.js : CFAv1Forwarder
```

Comment:
this is commented out, just want to double check that this is expected?

Reply:
this was indeed not intended, thx for spotting